sdk/contentsafety/azure-ai-contentsafety/src/main/java/com/azure/ai/contentsafety/models/ContentSafetyImageData.java

```diff
@@ -19,9 +20,8 @@ public final class ContentSafetyImageData {
     /*
      * The Base64 encoding of the image.
      */
-    @Generated
     @JsonProperty(value = "content")
-    private byte[] content;
+    private BinaryData content;
```

Comment:
I am not sure if this is the expected solution.

If it were me, I will keep this `byte[] content`. And do conversion in setter and getter.

But if all your test is good (please run a live one), you can do whatever you wish.

Reply:
The thing is the encoding of bytes. If it be `byte[]`, SDK knows it is base64 encoded as string, and will decode it. But if it is BinaryData, no one knows whether it is string, or bytes base64 encoded as string.

Reply:
So, you prefer we keep the generated code as byte[] instead of changing it to BinaryData?

Reply:
I prefer you keep the class var as `byte[]`, and only change the setter and getter method.

Reply:
updated